calicoctl/calico_ctl/diags.py

```diff
@@ -135,9 +145,11 @@ def save_diags(log_dir):
             f.writelines("Error running ipset: %s\n" % e)
 
     # Ask Felix to dump stats to its log file - ignore errors as the
-    # calico-node might not be running
-    subprocess.call(["docker", "exec", "calico-node",
-                     "pkill", "-SIGUSR1", "felix"])
+    # calico-node might not be running.  If using rkt runtime, skip this step, 
+    # since we can't yet run commands inside a rkt fly pod.
```

Comment:
We could just `pkill -SIGUSR1 felix` from the host? And that would work on docker too